scripts/update-helm-chart-reference/main.sh

```diff
@@ -15,11 +15,14 @@ docker run --rm \
 
 for file in ${DESTINATION}/*.md; do
   if grep -q '\--&gt;' $file; then
-    sed -i.bak 's/\&lt;.*\&gt;//g' $file
+    # Remove HTML comment lines
+    perl -i -pe "BEGIN{undef $/;} s/\&lt;.*\&gt;//gsm" $file
+    # Remove empty lines
+    awk 'NF > 0 || NR == 1 {blank=0} NF == 0 {blank++} blank < 2 {print}' $file > $file
+    # Remove trailing whitespace
+    sed -i 's/[ \t]*$//' $file
```

Comment:
What is this change about?

Reply:
ups that one was unintentionally copied from helm chart configuration work I am doing in parallel